tests/plugins/richcombo/manual/ariaexpanded.html

```diff
@@ -0,0 +1,11 @@
+<h2>Classic editor</h2>
+
+<textarea id="editor">
+	<p>I'm CKEditor 4 instance.</p>
+</textarea>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		lang: 'en'
+	} );
+</script>
```

Comment:
This test could be probably ignored for mobiles? I'm also wondering if it makes sense to enable it only for modern browsers, so not IE. WDYT?